apache/couchdb-fauxton
# Doc Editor Redux refactoring

## Overview

Refactoring of Doc Editor components to use Redux.

## Testing recommendations

- Edit a doc and save the changes. Open the editor again to verify the changes were made.
- Upload an attachment and verify the attachment shows up under the 'View Attachments' menu.
- Clone a doc. After cloning the editor loads up the cloned doc.
- Click the Delete button then confirm the deletion in the dialog. Verify the doc is no longer listed in the docs list.

## Checklist
